src/sentry/integrations/analytics.py

```diff
@@ -13,6 +13,16 @@ class IntegrationAddedEvent(analytics.Event):
     )
 
 
+class IntegrationDisabledNotified(analytics.Event):
+    type = "integration.disabled.notified"
+
+    attributes = (
+        analytics.Attribute("organization_id"),
+        analytics.Attribute("redis_key"),
```

Comment:
Do we need the redis key? If we have the provider and org ID that should be enough for us to tell which integration is disabled and for whom.